tests/unit/ops/transformer/inference/test_bias_gelu.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import torch
+import deepspeed
+from deepspeed.ops.op_builder import InferenceBuilder
+
+if not deepspeed.ops.__compatible_ops__[InferenceBuilder.NAME]:
+    pytest.skip("Inference ops are not available on this system",
+                allow_module_level=True)
+
+inference_module = InferenceBuilder().load()
```

Comment:
This will invoke an op build which will initialize cuda, i think this is the core of the test failures. Let's set this to `None` here and then build it only if `inference_module is None` inside each of the `def test_*` methods?

Reply:
@cmikeh2 -- please see the comment above. I think this is the problem.

Reply:
@jeffra -- thank you! Pushed a possible fix. Please check.